src/index.html

```diff
@@ -4,9 +4,318 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#Home" class="header__logo">
+        <img src="../src/images/logo.png" alt="logo-moyo">
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__ul">
+          <li>
+            <a href="#apple" class="header__link header__link--is-active is-active">apple</a>
+          </li>
+
+          <li>
+            <a href="#samsung" class="header__link">samsung</a>
+          </li>
+
+          <li>
+            <a href="#smartphones" class="header__link">smartphones</a>
+          </li>
+
+          <li>
+            <a
+              href="#laptops&computers"
+              class="header__link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li>
+            <a href="#gadgets" class="header__link">gadgets</a>
+          </li>
+
+          <li>
+            <a href="#tablets" class="header__link">tablets</a>
+          </li>
+
+          <li>
+            <a href="#photo" class="header__link">photo</a>
+          </li>
+
+          <li>
+            <a href="#video" class="header__link">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main-page">
+      <div class="main-page__wrapper">
+        <div class="card main-page__card" data-qa="card">
```

Comment:
```suggestion
        <div class="card catalog__card" data-qa="card">
```
It will me more informative